src/Main.elm

```diff
@@ -71,13 +87,22 @@ type Pun
     = Pun String
 
 
+type alias Debouncer =
+    { value : String
+    , timeout : Float
```

Comment:
Given you're only using one debouncer, what are your thoughts on moving this outside of here - either as a global value, or hard-coding when you sleep the process?